src/logging.rs

```diff
@@ -0,0 +1,51 @@
+use std::io;
+use super::*;
```

Comment:
do we need to include everything from the super module here?

Reply:
Found this way of wokring with includes in other crates: but can't find any reason for not just include what is realy needed...so yes lets change that!